docs/Manual/03 Schedules.md

```diff
@@ -16,39 +16,40 @@ for i in range(16):
+*Iteration space slices* is an abstract concept that affects different aspects of the Accera programming model. Since the iteration space dimensions are ordered, each element of the iteration space can be identified by a vector of coordinates.  For example, the vector (5, 6, 7) identifies the iteration at position 5 along the first dimension, position 6 along the second dimension, and position 7 along the third dimension. If one or more coordinates are replaced with the *wildcard* symbol `\*`, we get an *iteration space slice*, which is a set of iterations obtained by replacing the wildcard with all possible values. For example, (\*, \*, 5) represents a slice containing all the elements with 5 as their last coordinate. The *dimension of a slice* equals the number of wildcards in its definition.
 
 ### Loops, indices, and dimensions
-When we defined `nest`, we used variables such as `i`, `j`, `k` to name the loops in the loop-nest. When we described the default schedule using equivalent for-loops, `i`, `j`, and `k` became the index variables of those loops. When we represent a schedule as an iteration space, these variables are used as the names of the corresponding iteration space dimensions. From here on, we move seamlessly between these different representations and use the terms *loop*, *index*, and *dimension* interchangeably.
+When we defined `nest`, we used variables such as `i`, `j`, and `k` to name the loops in the loop-nest. When we described the default schedule using equivalent for-loops, `i`, `j`, and `k` became the index variables of those loops. When we represented a schedule as an iteration space, these variables were used as the names of the corresponding iteration space dimensions. From here on, we move seamlessly between these different representations and use the terms *loop*, *index*, and *dimension* interchangeably.
 
 ## Schedule transformations
-Iteration space transformations change the shape of the iteration space, possibly adding dimensions and padding the space with empty elements.
+Iteration space transformations change the shape of the iteration space, possibly by adding dimensions or padding the space with empty elements. 
 
-The iteration space always retains its rectilinear shape (i.e., the shape of a hypercube). In some cases, Accera transformations must pad the iteration space with empty elements to avoid reaching a jagged iteration space structure.
+The iterations space always retains its rectilinear hypercube shape. In some cases, Accera transformations must pad the iteration space with empty elements to avoid reaching a jagged iteration space structure.
```

Comment:
"iteration space" is a term (yeah, we need a glossary... :))

Reply:
We do need it :(